package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "alloy",
-  "version": "1.18.0",
+  "version": "2.0.2",
```

Comment:
Not sure why this was still on 1.x, but now it should be clean!

Reply:
The package version is also in the package-lock, but when we manually bump the version in GitHub, we don't bump it in the lock file.